Dockerfile.agent

```diff
@@ -8,7 +8,7 @@ ARG VERSION
 RUN apt-get update && apt-get install build-essential -y
 
 RUN pip install prometheus-client
```

Comment:
Missed one. Why is this separate anyway?

Reply:
since we don't want to add `prometheus-client` to the default flytekit dependencies.